extensions/azure-keyvault/deployment/src/main/java/io/quarkiverse/azure/keyvault/secret/deployment/KeyVaultSecretProcessor.java

```diff
@@ -37,6 +35,11 @@ AdditionalBeanBuildItem producer() {
         return new AdditionalBeanBuildItem(KeyVaultSecretClientProducer.class);
     }
 
+    @BuildStep
+    public void azureKeyVaultSecretConfigFactory(BuildProducer<RunTimeConfigBuilderBuildItem> runTimeConfigBuilder) {
```

Comment:
can we remove the `public` here?

Reply:
Good point. Many of the other `BuildStep` annotated methods are not `public`.

Reply:
I support this one.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/266/commits/57f60787beb187d26e7b65e7255c3215ac13180f